lib/autsr.gi

```diff
@@ -666,7 +730,7 @@ condcnt:=0;
     #if IsNormal(sub,Aperm) and IndexNC(sub,Aperm)<=50000 then
     #  sub:=SubgroupConditionAbove(sub,cond,Aperm);
     #else
```

Comment:
So I guess that means the commented out code above can go away?

Reply:
Yes.